dev/freestanding/femr/docker-compose.dev.hydrant.yaml

```
@@ -0,0 +1,12 @@
+# docker-compose development override for hydrant service
+---
+version: "3.7"
+services:
+  hydrant:
+    command: sh -c "cd /opt/app && flask run --host 0.0.0.0 --port ${PORT:-8000}"
```

Our WORKDIR is `/opt/app` by default (in the [Dockerfile](https://github.com/uwcirg/hydrant/blob/develop/Dockerfile#L3)) so the `cd` shouldn't be necessary
```suggestion
    command: sh -c "flask run --host 0.0.0.0 --port ${PORT:-8000}"
```